src/input/manager.rs

```diff
@@ -1210,11 +1210,26 @@ impl Manager {
                         }
                     };
 
-                    if !is_bluetooth {
+                    // Some virtual gamepads we DO want to manage
+                    const VIRT_DEVICE_WHITELIST: &[&str] = &[
```

Comment:
I there somewhere else we can keep this const, such as the top of the file or in udev.rs? Doing it inline here will make it harder to keep track of to add more and it is only a few bytes.